analysers/analyser_osmosis_indoor.py

```diff
@@ -26,20 +26,53 @@
 sql00 = """
 CREATE TEMP TABLE indoor_surfaces AS
 SELECT
-    id,
-    linestring AS geom,
-    nodes,
-    tags->'indoor' AS indoor,
-    tags->'level' AS level,
-    tags,
-    (NOT tags?'access' OR NOT tags->'access' IN ('no', 'private')) AS public_access
+    ways.id,
+    ways.linestring AS geom,
+    ways.nodes,
+    ways.tags->'indoor' AS indoor,
+    ways.tags->'level' AS level,
+    ways.tags,
+    (NOT ways.tags?'access' OR NOT ways.tags->'access' IN ('no', 'private')) AS public_access
 FROM
     ways
 WHERE
-    is_polygon AND
-    tags != ''::hstore AND
-    tags?'indoor' AND
-    tags->'indoor' in ('room', 'corridor', 'area', 'level')
+    ways.is_polygon AND
+    ways.tags != ''::hstore AND
+    ways.tags?'indoor' AND
+    ways.tags->'indoor' in ('room', 'corridor', 'area', 'level')
+UNION ALL
+SELECT
+    relations.id,
+    ST_MakePolygon(outer_ways.linestring,array_agg(inner_ways.linestring)) AS geom,
+    ARRAY(select unnest(array_agg(inner_ways.nodes))) ||outer_ways.nodes as nodes,
```

Comment:
```suggestion
    ARRAY(select unnest(array_agg(inner_ways.nodes))) || outer_ways.nodes as nodes,
```

Reply:
`ARRAY(select unnest(array_agg(inner_ways.nodes)))` realy need `ARRAY(select unnest())` as `array_agg` is already an array.